docs/writing-policies/go/02-scaffold.md

```diff
@@ -26,7 +27,7 @@ constrained_labels:
   cost-center: "^cc-\d+"
```

Comment:
Why changing this regex? It looks like a typo.

Reply:
I didn't. What should it be?

Reply:
Never mind, there is no change. It just the syntax highlight that looks like a removal for me.